modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/java/type/GenericInfo.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.module.extension.internal.loader.java.type;
+
+import java.util.List;
+
+/**
+ * Describes de generics of a Type
+ *
+ * @since 4.1
+ */
+public class GenericInfo {
```

Comment:
this is not a generic info, but something that contains generic infos and relates it to a type